app/components/osf-footer/component.ts

```diff
@@ -0,0 +1,12 @@
+import Component from '@ember/component';
+import config from 'ember-get-config';
+import { serviceLinks } from 'ember-osf-web/const/service-links';
+
+export default class OsfFooter extends Component {
+    serviceLinks = serviceLinks;
+
+    constructor() {
+        super();
+        Object.defineProperty(this, 'supportEmail', { value: config.support.supportEmail });
```

Comment:
Is it important that this not be writable, or could we do a normal property `supportEmail = config.support.supportEmail;`?